src/chromo/models/__init__.py

```diff
@@ -35,3 +36,89 @@
     "Pythia6",
     "Pythia8",
 )
+
+
+# This is a workaround of a problem with "fixed" wheels
+# which is produced by "delocate" package on cibuildwheels
+# It changes Python -> the path of libpython in the "*.so" files
+# using standard "otool" and "install_name_tool" routines
+def fix_macos_installation(logfile):
+    import subprocess
+    import re
+    from pathlib import Path
+    from distutils.sysconfig import get_config_var
+
+    models_dir = Path(__file__).parent
+    for ext_file in models_dir.glob("*.so"):
+        ext_file = str(ext_file)
+
+        # Get libraries required by extension module
+        cmd = ["otool", "-L", ext_file]
```

Comment:
Add a short line in the README that installing on mac requires command line utils `xcode-select --install`

Reply:
The info about it is added to README.